ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -2,33 +2,33 @@
 import os
 import uuid
 
+from dcos_installer import backend
+
 
 class InstallerAnalytics():
-    def __init__(self, config):
-        self.uuid = self.new_uuid()
-        self.bootstrap_version = os.getenv("BOOTSTRAP_ID")
-        self.customer_key = config["customer_key"] if 'customer_key' in config else ""
-        self.source = "onprem"  # If you're using the installer you're not in AWS or Azure
+    def __init__(self):
+        self.uuid = str(uuid.uuid4())
 
     def send(self, action, install_method, num_errors):
         """Sends analytics track data to segmentIO.
         variant: string | open or enterprise
         action: string | preflight, deploy, or postflight
         install_method: string | gui, cli or advanced
         """
-        analytics.write_key = "39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP"
+        analytics.write_key = "51ybGTeFEFU1xo6u10XMDrr6kATFyRyh"
 
-        analytics.track(user_id=self.customer_key, anonymous_id=self.uuid, event=action, properties={
-            "install_id": self.uuid,
-            "bootstrap_id": self.bootstrap_version,
-            "provider": self.source,
+        # We set customer key from config to avoid loading the config during class init
+        customer_key = backend.get_config().get("customer_key", None)
+
+        analytics.track(user_id=customer_key, anonymous_id=self.uuid, event=action, properties={
+            "provider": "onprem",
             "source": "installer",
+            "variant": os.getenv("BOOTSTRAP_VARIANT"),
```

Comment:
My thinking was this code will never run unless the CLI is working, and the CLI already has a check on this ENV var. That being said, better safe than sorry.